templates/install_rvm.sh.erb

```diff
@@ -3,12 +3,16 @@
 USER=<%= @user %>
 
 SERVERS=(
-    hkp://keys.gnupg.net
     hkp://pgp.mit.edu
+    hkp://keys.gnupg.net
 )
 
 for server in "${SERVERS[@]}"
 do
-    su -c "gpg --keyserver ${server} --recv-keys D39DC0E3" - $USER
+    success=`su -c "gpg --keyserver ${server} --recv-keys D39DC0E3" - $USER`
+
+    if $success; then
+        break
+    fi
```

Comment:
would it make more sense to check the return value of the command?  ($?)

Testing locally i still hit break even if success is empty (implying no output)